Src/BootCamp.Chapter/Program.cs

```diff
@@ -9,6 +9,122 @@ class Program
     {
         static void Main(string[] args)
         {
+            // Need to refactor the code I made in Lesson 2.
+            // Use functions to achieve DRY (as little duplicate code as possible).
+
+            // Name input
```

Comment:
Comments can be removed, the functions speak for themselves.

Reply:
Comments have been removed.